docs/enhancements/cluster_pool_hot_spares.md

```diff
@@ -0,0 +1,171 @@
+
+If the ClusterPool has `hibernateAfter` set, that clock starts on a given ClusterDeployment from the time its
+[Hibernating condition is Running](https://github.com/openshift/hive/blob/28f67bab3dac5fb9dec405e0536d64187faf964d/pkg/controller/hibernation/hibernation_controller.go#L240-L244);
+or, if the status of that condition is Unknown, from
+[when installation completed](https://github.com/openshift/hive/blob/28f67bab3dac5fb9dec405e0536d64187faf964d/pkg/controller/hibernation/hibernation_controller.go#L229-L239).
+If we don't account for this, Running unassigned CDs in the pool will hibernate, and then the [new logic](#decouple-powerstate-on-creation) will immediately wake them back up, resulting in unnecessary churn and potential claim delays.
+So we'll:
+- Add another Condition to ClusterDeployment called `Claimed`, indicating claim assignment status.
+  It is:
+  - `Unknown` for CDs not part of a pool, permanently.
+  - Initialized and maintained as `False` for unassigned pool CDs.
+  - Set to `True` when a CD is assigned to a claim. This should be terminal: once `True`, it should never change again.
+- Add logic to the hibernation controller as follows:
+  - If the CD is part of a ClusterPool
+    - If it is unclaimed, ignore `hibernateAfter`* -- never hibernate.
+    - If it is claimed, use as the base time the _later_ of
+      - the `lastTransitionTime` of the new `Claimed` Condition
+      - the existing algorithm's base time (this accounts for hibernating/resuming during the normal post-claim lifecycle of the CD).
+  - Else (not part of a ClusterPool) do what we do today.
+
+*This also means a manually-resumed unclaimed CD from a pool will never hibernate at the behest of `hibernateAfter`.
+As noted [above](#clusterclaim-assignment-by-creationtimestamp), you shouldn't be trying to manage the PowerState of pool-owned CDs manually.
+This goes double if you're using `hibernateAfter` with ClusterPools.
+
+### A Note About Tuning
+
+Proper tuning will vary wildly depending on the consumer, of course.
+We're giving them the knob to tune with.
+As a starting point, perhaps:
+```
+runningCount = round_up(claim_frequency * install_time)
+```
+For example, if your pool sees one claim every 15 minutes, and it takes 40 minutes to install a cluster:
+```
+runningCount = round_up(1/15 * 40) = 3
+```
+Further tuning would depend on e.g.
+- How "smoothly" claims come in. If your average of 1/15m results from 11 per hour from 9a-5p and 8 total overnight, maybe you want to use the 11/h (`runningCount = 8`) to ensure you can handle peak demand.
+- Whether your priority is cost savings or responsiveness. If the former, perhaps you want to lean towards the 0.5/h (`runningCount = 1`) -- or not use a `runningCount` at all.
+- How much it "costs" you to retune the pool frequently. E.g. if you have the means for automation and a predictable usage curve, perhaps you configure a job to set `runningCount = 8` at 8:50am M-F and `runningCount = 0` at 5p M-F.
```

Comment:
A histogram metric tracking time from claim request to fulfillment would be great to see in heavily loaded envs, and would help inform if you've got enough of a running count or not.

Reply:
I like it.

Reply:
...but it seems only partially related to this feature, so I'd like to keep it separate. I've opened HIVE-1643 to track it.